### PR TITLE
Gb/upload captured photo

### PR DESCRIPTION
**Pull request by marecar3**
**Description :**
This PR introduces upload of captured image from camera through Gutenberg Editor which implementation is based on this [issue](https://github.com/wordpress-mobile/gutenberg-mobile/issues/206).

**Following PR's :**
[gutenberg-mobile 
](https://github.com/wordpress-mobile/gutenberg-mobile/pull/492)[gutenberg](https://github.com/WordPress/gutenberg/pull/13450)

ezgif com-video-to-gif 2

### CODE REVIEW

File: libs/editor/WordPressEditor/src/main/java/org/wordpress/android/editor/GutenbergEditorFragment.java [hunk 1]
```diff
@@ -47,6 +52,10 @@
 
     private static boolean mIsToolbarExpanded = false;
 
+    private static final String[] sPermissions = new String[]{Manifest.permission.CAMERA,
```
Comment (mzorz) [inline libs/editor/WordPressEditor/src/main/java/org/wordpress/android/editor/GutenbergEditorFragment.java]: Let's rename this variable here to something more descriptive like `sCameraAndExternalStoragePermissions`

File: libs/editor/WordPressEditor/src/main/java/org/wordpress/android/editor/GutenbergEditorFragment.java [hunk 2]
```diff
@@ -184,6 +203,33 @@ public void run() {
         return view;
     }
 
+    private void requestCameraandStoragePermissions() {
```
[the remark this answers is not in the document]
Comment (marecar3) [reply libs/editor/WordPressEditor/src/main/java/org/wordpress/android/editor/GutenbergEditorFragment.java]: Fixed.

File: libs/editor/WordPressEditor/src/main/java/org/wordpress/android/editor/GutenbergEditorFragment.java (hunk 1 above)
Comment (marecar3) [reply libs/editor/WordPressEditor/src/main/java/org/wordpress/android/editor/GutenbergEditorFragment.java]: Fixed.

File: libs/editor/WordPressEditor/src/main/java/org/wordpress/android/editor/GutenbergEditorFragment.java [hunk 3]
```diff
@@ -117,6 +126,16 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle sa
                     public void onUploadMediaButtonClicked() {
                         mEditorFragmentListener.onAddPhotoClicked();
                     }
+
+                    @Override
+                    public void onCapturePhotoButtonClicked() {
+                        if (!hasPermissions(sPermissions)) {
```
[the remark this answers is not in the document]
Comment (marecar3) [reply libs/editor/WordPressEditor/src/main/java/org/wordpress/android/editor/GutenbergEditorFragment.java]: Fixed. thanks!

**Pull request merged**